cu-webteam/d8-platform
# acsf init

# Description

Modifications from running `blt recipes:acsf:init:all`
This process modifies:
* blt/blt.yml
* settings.php
* all factory hooks
* .htaccess

## Related PRs

None

## Todos

N/A

## Deploy Notes

## Steps to Test or Reproduce

These things need to be checked after deploy to production:
* db-update hook - check that config import runs and the notification is pushed to slack about the code update
* build a site on prod to test
    * post-install hook successfully modifies SSO config (successful login from ACSF on new site)
    * build a site on prod and check logs (site should not be created with warning, should have no 'APC cache failed to clear' errors
